eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/JdbcEventStorageEngine.java

```diff
@@ -764,6 +835,356 @@ public void setGapCleaningThreshold(int gapCleaningThreshold) {
         private int gapCleaningThreshold = DEFAULT_GAP_CLEANING_THRESHOLD;
         private boolean extendedGapCheckEnabled = DEFAULT_EXTENDED_GAP_CHECK_ENABLED;
 
+        private CreateTokenAtStatementBuilder createTokenAt = (connection, dateTime) -> {
```

Comment:
As just discussed, it would be cleaner to move these lambdas to a utility class as static functions.

Reply:
As agreed, all the lambdas are now on a utility class.